app/services/operations/create_application.rb

```diff
@@ -0,0 +1,28 @@
+require 'laa_crime_schemas'
+
+module Operations
+  class CreateApplication
+    attr_reader :payload
+
+    def initialize(payload:)
+      @payload = payload
+
+      validate!
+    end
+
+    def call
+      app = CrimeApplication.create!(application: payload)
+
+      { id: app.id, status: :created }
```

Comment:
This payload is not currently used anywhere on Apply after an application has been POSTed, although the API client propagates it. I think to avoid confusion maybe we can return just the `{ id: app.id }` and not `status` (because this status was supposed to be the application status, i.e. `submitted` and not the http status that is handled instead at the grape level).

Reply:
Yes I agree. Will remove.